app/src/main/java/com/rob729/quiethours/Adapter/ProfileListAdapter.kt

```diff
@@ -63,7 +65,15 @@ class ProfileListAdapter(
             Color.rgb(255, 143, 0),
             Color.rgb(216, 27, 96)
         )
-
+        private fun SnackBar(cardView: CardView, message: String) {
```

Comment:
Move the function to Utils and take view, message,  as parameter.
Don't take cardView type as a paramter.

Reply:
Done

Reply:
My bad @coder2699. I forgot to mention that snackbar length was also supposed to be taken.
Make the default value of snackbar length to short.
Also, use this function in MainFragment(in swipe to delete) and NewProfileFragment